keva/server/src/main/java/com/jinyframework/keva/server/ServiceFactory.java

```diff
@@ -6,32 +6,35 @@
 import com.jinyframework.keva.server.core.ConnectionServiceImpl;
 import com.jinyframework.keva.server.core.SnapShotServiceImpl;
 import com.jinyframework.keva.server.core.SnapshotService;
+import lombok.Setter;
 
+@Setter
 public final class ServiceFactory {
```

Comment:
Can we remove all null checks, synchronized and manually set implementations in an init phase?